string obj_label  
geometry_msgs/PoseWithCovariance posecov
float32 score
int32 mask_pixel_count
ros_cosy_tracking/BoundingBox bbox